src/tic_tac_toe/referee.clj

```diff
@@ -11,3 +11,36 @@
 (defn validate-move
   [player move]
   (convert-to-spot move))
+
+(defn draw?
+  [board]
+  (board/full? board))
+
+(defn matching-pattern?
+  [token pattern]
+  (loop [token token
+         pattern pattern]
+    (match pattern
+      [token] true
+      [" " & r] false
```

Comment:
What does `" "` mean? Why is it important?

Reply:
[Updated this](https://github.com/saralein/tic-tac-toe_clojure/pull/6/files#diff-f014d0e7db79dee6fc42af72f74d5325R19).  I added an additional function to handle the idea of [empty patterns](https://github.com/saralein/tic-tac-toe_clojure/pull/6/files#diff-f014d0e7db79dee6fc42af72f74d5325R33).